docker-autoscale/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker-private.infra.cloudera.com/cloudera_base/cldr-java:11.0.6-cldr1-jdk-slim-buster
+FROM docker-private.infra.cloudera.com/cloudera_base/cldr-java:11.0.13-cldr-jre-slim-buster-15122021
```

Comment:
Would it be possible to add some hint as comments, about why we are not using alpine?